### ISSUE TITLE
How to build aruco(opencv_contrib3.3) for android with cmake-gui

### ISSUE THREAD

**Issue by 44326334** [richdyang/opencv3-android-sdk-with-contrib#1]
I saw your answer on github of opencv_contrib. While I success build opencv_contrib for Java and working. but I don't know how to build it for android . I installed NDK(android-ndk-r14b) and not to saw any parameter of cmake-gui for android. please help me. thanks!

**Comment by 44326334** [comment]
Now I use your project of opencv_contrib3.3.1 for test

**Comment by richdyang** [comment]
@44326334 Have you read my blog post here?  http://yangchao.me/post/build-opencv-android-sdk/

**Comment by 44326334** [comment]
@richdyang yes,I readed. my project depend on openCV3.3.0 and openCV_contrib. so first I used your builded(3.3.1 version) into my project. if or not, I must build opencv3.3.0 version for myself. but Now I `Import new module` into your java folder and setting `gradle` and project's `settings`. then I create a new Mat to put a byte[] data. the AS throw a except `unsatisfiedlinkerror` error. so I don't know how to work for import your aruco version 3.3.1 into my project.
